release-notes/5.0/api-diff/netcoreapp3.1/.Net/5.0_System.Formats.Asn1.md

```diff
@@ -0,0 +1,222 @@
+# System.Formats.Asn1
+
+``` diff
++namespace System.Formats.Asn1 {
++    public readonly struct Asn1Tag : IEquatable<Asn1Tag> {
```

Comment:
Everything in the System.Formats.Asn1 is in a NuGet package available to .NET Standard 2.0; so it's available with netcoreapp3.1.

On the one hand, I like that people who can benefit from it it can see that it exists.  On the other, I'm not sure it really belongs in this diff.

Reply:
I suggest we add comments or text at the top of the file to indicate places where APIs were previously available via nuget and are now available in-box.

Reply:
I was going to say "but System.Formats.Asn1 isn't in the ref-set for net5.0, it's just in the shared framework as an implementation detail".  But, apparently that's not true, it seems to be in the ref-set.